src/js/helpers/ResourcesHelpers.js

```diff
@@ -526,4 +526,38 @@ export function getMissingResources() {
       });
     }
   });
+  getMissingUnzippedResources('en/lexicons'); // TODO: this is temporary - eventually this will be packaged in catalog
+}
+
+/**
+ * copy an unzipped folder if it does not exist
+ * @param {string} subPath - resource subpath to copy
+ */
+export function getMissingUnzippedResources(subPath) {
```

Comment:
This function does not follow the DRY principle. it's basically a copy of the code in getMissingResources() which means making unnecessary file systems (fs) calls. I would suggest moving the specific code that copies the missing files, to the getMissingResources() function.